djhtml/__main__.py

```diff
@@ -103,7 +103,10 @@ def main():
                     problematic_files += 1
                     _error(e)
                     continue
-                _info(f"reindented {output_file.name}")
+                if not options.quiet:
+                    _info(f"reindented {output_file.name}")
+        elif not options.quiet and filename != "-":
+            _info(f"{filename} would be reindented.")
```

Comment:
This will always print `{filename} would be reindented`, even when the file _would not_ have been reindented!